README.md

```diff
@@ -107,7 +107,7 @@ imageView.file = (PFFile *)someObject[@"picture"]; // remote image
 
 ## Learn More
 - Check out [ParseUIDemo](https://github.com/ParsePlatform/ParseUI-iOS/tree/master/ParseUIDemo) project
-- Read the [iOS Guides](https://parse.com/docs/ios_guide#ui/iOS)
+- Read the [iOS Guides](http://parseplatform.github.io/docs/ios/guide/)
```

Comment:
This link goes to the current iOS Guide. There is no UI section, so that portion of the URL was not included, as it did not do anything.